### ISSUE TITLE
Question about the cam_T_cam

### ISSUE THREAD

**Issue by Giperx** [fangzhou2000/DrivingForward#10]
Excellent work! 

I have a question about these codes:

https://github.com/fangzhou2000/DrivingForward/blob/5d0a2c7d6a6358f55318faac80e351cf75dff032/models/drivingforward_model.py#L201-L207

When I run Evaluation using `python -W ignore eval.py --weight_path ./weights_SF --novel_view_mode SF`, It means that the final render output images don't use the prediction of pre-trained PoseNetwork but use `cam_T_cam`.
The `cam_T_cam` was read from dataset GT, but the paper says that the GT of the extrinsics is not required. Can you explain this? I am not very clear about this part.

Looking forward to your reply! Thank you!

**Comment by RenshengJi** [comment]
same question

**Comment by fangzhou2000** [comment]
Hi, our method does not require the GT pose **during training**, since the depth network and pose network learns metric depth in a self-supervised manner. At the inference stage, since our comparison methods use the GT pose, we also use the GT pose to align with them.

If you do not want to use the GT pose during inference, you can simply comment out the code that reads the GT pose and use the prediction pose instead.

Besides, the above code is just actually used for MF mode; if you evaluate in the SF mode, the pose between adjacent cameras is not used, since the relative pose of the camera is fixed within the same frame.
